tests/datavolume_test.go

```diff
@@ -314,7 +314,10 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				if args.checkPermissions {
 					// Verify the created disk image has the right permissions.
 					By("Verifying permissions are 660")
-					Expect(f.VerifyPermissions(f.Namespace, pvc)).To(BeTrue(), "Permissions on disk image are not 660")
+					Eventually(func() bool {
+						result, _ := f.VerifyPermissions(f.Namespace, pvc)
+						return result
+					}, timeout, pollingInterval).Should(BeTrue(), "Permissions on disk image are not 660")
 					err := utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
 					Expect(err).ToNot(HaveOccurred())
```

Comment:
this is a little odd, if this ever loops, we'll try to create a new pod but the previous one will still exist.

Reply:
I guess that's fine because the function already handles this gracefully
/lgtm